### ISSUE TITLE
v0.8.2: When you delete a QE Journal Entry with a Note, get an error from Foundry

### ISSUE THREAD

**Issue by spetzel2020** [spetzel2020/quick-encounters#49]
I suspect this is because in Foundry 0.8.2 it is automatically handling deletion of associated notes (see `_onDelete` below)

```
foundry.js:14575 Uncaught (in promise) TypeError: Cannot set property entry of #<Note> which has only a getter
    at JournalEntry._onDelete (foundry.js:14575)
    at ClientDatabaseBackend.callback (foundry.js:9050)
    at foundry.js:9002
    at Array.map (<anonymous>)
    at ClientDatabaseBackend._handleDeleteDocuments (foundry.js:9002)
    at ClientDatabaseBackend._deleteDocuments (foundry.js:8912)
    at async Function.deleteDocuments (document.mjs:402)
    at async JournalEntry.delete (document.mjs:463)
```

**Comment by spetzel2020** [comment]
The _onDelete only deletes notes in the current scene, so we still need the QE trigger to handle deletion notes in other scenes

**Comment by spetzel2020** [comment]
This is a generic Foundry 0.8.8 bug; the `Journal._on_Delete()` trigger tries to set `n.entry = null` but that is a getter only. Bugged in Foundry.

**Comment by spetzel2020** [comment]
Reopening to test again when Foundry bug is addressed (in 0.8.9?)

**Comment by spetzel2020** [comment]
In Foundry 0.8.9, the Journal Entry is now deleted correctly so the QE trigger needs to be in place.

**Comment by spetzel2020** [comment]
Released in [v0.8.5](https://github.com/spetzel2020/quick-encounters/commit/bf7f77344154fbc04ae3101d1704ecb13725c5a1)
